spec/helpers/service_provider_helper_spec.rb

```diff
@@ -146,6 +146,9 @@
         expect(config_hash(oidc_jwt_sp_2)).to include(attribute_name)
       end
     end
+    it 'returns the ial config as an integer instead of a string' do 
+      expect(config_hash(saml_sp_ial_2)['ial'].class).to eq(Integer)
```

Comment:
**nitpick:** I think you can use the assertion `be_an` to clean this up a little:
```suggestion
      expect(config_hash(saml_sp_ial_2)['ial']).to be_an(Integer)
```

Reply:
Yes.  I agree.